docs/encrypted-innodb-tablespace-backups.md

```diff
@@ -408,7 +426,7 @@ $ xtrabackup --prepare --target-dir=/data/backup \
 ```
 
 There are no `--keyring-vault...`,\`\`–keyring-file…\`\`,
-or `--component-keyring-file-config` options here,
+or `--component-keyring-config` options here,
```

Comment:
Everywhere in this doc, I find this option,  --keyring-vault-config I doesn't seem to be a valid option in 8.0 too.
@altmannmarcelo any idea ^ ?

for generated keys section, we have to update the section to use component variants.

Reply:
This PR is closed. That's why I have not updated the examples here. I fixed the examples here https://github.com/percona/pxb-docs/pull/185

Reply:
My bad, I see that they are fixed in 185.